10.0/s2i/bin/run

```diff
@@ -1,2 +1,12 @@
 #!/bin/bash
-exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+
+run_wildfly() {
+  if [ "$DEV_MODE" == true ]; then
+    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0 -Dhot.deploy.enabled=true -DENABLE_JPDA=true
+  else
+    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+  fi
+}
+
+run_wildfly
```

Comment:
flexible in what sense?  our best practices basically just say "configuration options should be exposed as env variables so users can control them at build/runtime"